content/swagger/resources/products_uuid/definitions/examples/products_uuid.yaml

```diff
@@ -26,6 +26,7 @@ x-SaaS: {
         "family": "tshirt",
         "groups": [],
         "parent": null,
+        "root_parent": "the_code_of_the_root_product_model",
```

Comment:
I think it's better to find a example more realistic. May be you can change the name to "red top XL" for example and rename the root model to «top»